tests/vector_tools/interpolate.cc

```diff
@@ -0,0 +1,282 @@
+#include <deal.II/base/utilities.h>
```

Comment:
can you add the usual copyright header to this test?

Reply:
Copyright added!